code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -333,3 +333,209 @@ BONUS
+		armor = min(20, A.resistance)
+	if(A.transmission >= 6) //higher damage
+		Power = 1.4  //the typical +100% is waaaay too strong here when the symptom is stacked. +40% is sufficient
+
+/datum/symptom/spiked/Activate(var/datum/disease/advance/A)
+	var/mob/living/carbon/human/H = A.affected_mob
+	if(!..())
+		return
+	switch(A.stage)
+		if(1)
+			if(prob(base_message_chance))
+				to_chat(H, "<span class='warning'>You feel goosebumps pop up on your skin.</span>")
+		if(2)
+			if(prob(base_message_chance))
+				to_chat(H, "<span class='warning'>Small spines spread to cover your entire body.</span>")
+		if(3)
+			if(prob(base_message_chance))
+				to_chat(H, "<span class='warning'> Your spines pierce your jumpsuit.</span>")
+		if(4, 5)
+			if(!done)
+				H.AddComponent(/datum/component/spikes, 5*power, armor, A.GetDiseaseID()) //removal is handled by the component
+				to_chat(H, "<span class='warning'> Your spines harden, growing sharp and lethal.</span>")
+				done = TRUE
+			if(H.pulling && iscarbon(H.pulling)) //grabbing is handled with the disease instead of the component, so the component doesn't have to be processed
+				var/mob/living/carbon/C = H.pulling
+				var/def_check = C.getarmor(type = "melee")
+				C.apply_damage(1*power, BRUTE, blocked = def_check)
+				C.visible_message("<span class='warning'>[C.name] is pricked on [H.name]'s spikes.</span>")
+				playsound(get_turf(C), 'sound/weapons/slice.ogg', 50, 1)
+			for(var/mob/living/carbon/C in ohearers(1, H))
+				if(C.pulling && C.pulling == H)
+					var/def_check = C.getarmor(type = "melee")
+					C.apply_damage(3*power, BRUTE, blocked = def_check)
+					C.visible_message("<span class='warning'>[C.name] is pricked on [H.name]'s spikes.</span>")
+					playsound(get_turf(C), 'sound/weapons/slice.ogg', 50, 1)
+
+
+/datum/symptom/pustule
+	name = "Bubonic Infection"
+	desc = "The virus causes festering infections in the host's lymph nodes, leading to festering buboes that deal toxin damage."
+	stealth = -1
+	resistance = -2
+	stage_speed = 2
+	transmission = 1
+	level = 6
+	severity = 3
+	symptom_delay_min = 20
+	symptom_delay_max = 60
+	prefixes = list("Pestilent ", "Bubonic ")
+	var/pustules = 0
+	var/shoot = FALSE
+	threshold_desc = "<b>Transmission 4:</b>Buboes will occasionally burst when disturbed or left too long, shooting out toxic pus.<br>\
+					<b>Transmission 6:</b> Pustules appear on the host more frequently, dealing more damage."
+
+/datum/symptom/pustule/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.transmission >= 6)
+		severity += 1
+	if(A.transmission >= 4)
+		severity += 1
+		prefixes = list("Ballistic ", "Pestilent ", "Bubonic ")
+
+/datum/symptom/pustule/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.transmission >= 4)
+		shoot = TRUE
+	if(A.transmission >= 6)
+		power += 1
+	if(iscarbon(A.affected_mob))
```

Comment:
Same comment as earlier regarding carbon checks; sometimes you check if it's a carbon, sometimes you don't.

Reply:
ditto to above